Main/Source/char.cpp

```diff
@@ -5745,6 +5749,23 @@ int character::GetRandomNonVitalBodyPart() const
   return OKBodyParts ? OKBodyPart[RAND() % OKBodyParts] : NONE_INDEX;
 }
 
+int character::GetTotalCharacterWeight() const
+{
+  int CharacterWeight = 0;
+
+  for(int c = 0; c < BodyParts; ++c)
+  {
+    bodypart* BodyPart = GetBodyPart(c);
+
+    if(BodyPart)
+    {
+      Weight += BodyPart->GetWeight();
```

Comment:
`Weight` should be `CharacterWeight` here. Other than that looks good to me! 👍

Reply:
Oops, thanks.